code/src/components/LoadingIndicator.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import { useSelector } from "react-redux";
+
+import styled from "styled-components/macro";
```

Comment:
styled-components is an outer dependency, therefore it should be placed next to react.